infrateam_first_responder.md

```diff
@@ -2,161 +2,106 @@
-* Schedule:  https://docs.google.com/spreadsheets/u/1/d/13TJR93Yc9_eF5B7w4XDx6ggG_wb3aLkgCHjpLwmHPBA/
+  * E.g., if Alice and Bob are up next week, `/remind #dlss-infrastructure "@alice is the first responder week of January 8, and @bob is on deck" Monday at 3 am`
 
-## Proactively Check for Production Problems
-
-See [How to Proactively Check for Production Problems](#how-to-proactively-check-for-production-problems) section below for specifics.
+### Monitor Honeybadger (at least once per day)
+Monitor [Honeybadger alerts](https://app.honeybadger.io/projects) for projects in the Infrastructure portfolio
 
-## Triage Production Problems
-
-If a user reports a problem, or if one is surfaced from monitoring, the first responder ought to timebox an investigation of the problem (_start with 30 min?_). See [How to Triage Production Problems](#how-to-triage-production-problems) section below for more specifics.
+### Monitor Slack Channels (at least once per day)
+Slack channels relevant to applications in the portfolio:
+  - `#dlme` - "Digital Library of the Middle East" - we are responsible for dlme-transform, related to indexing of materials.
+  - `#dlss-aaas` - "Accessioning as a Service" - where accessioneers may surface problems
+  - `#dlss-etds-dev` - Electronic Theses and Dissertations channel
+  - `#dlss-infrastructure` <-- our team's main channel
+  - `#dlss-preservation-dev`
+  - `#dlss-sinopia-dev` - Sinopia is our linked data channel
+  - `#earthworks` - for gis-robot-suite related discussion
+  - `#sdr-operations`
+  - `#sul-cap-collab` - has developers in the School of Medicine working on the Profiles project (which we connect to with our sul-pub system)
+  - `#web-archiving`
 
-## Improve Troubleshooting Documentation as Needed
+### Check Your Email (at least once per day)
+Address any issues from user feedback email lists, cron daemon emails, etc.
+
+### Monitor PresCat Dashboard (at least once per week)
+Anything appearing in red in the [PresCat Dashboard](https://preservation-catalog-web-prod-01.stanford.edu/dashboard) should be addressed.
+
+### Monitor Honeybadger checkins (at least once per week)
+All missing production checkins should be addressed. The following are current checkins:
+* PresCat: https://app.honeybadger.io/projects/54415/check_ins
+* Sul-Pub: https://app.honeybadger.io/projects/50046/check_ins
+* H2: https://app.honeybadger.io/projects/77112/check_ins
+* WAS Robot Suite: https://app.honeybadger.io/projects/51141/check_ins
+* DSA: https://app.honeybadger.io/projects/50568/check_ins
+* SDR API: https://app.honeybadger.io/projects/67994/check_ins
+* Preassembly: https://app.honeybadger.io/projects/52900/check_ins
+* Workflow: https://app.honeybadger.io/projects/58890/check_ins
+* WAS Registrar App: https://app.honeybadger.io/projects/63547/check_ins
+* Google Books: https://app.honeybadger.io/projects/67990/check_ins
+* Hydra ETD: https://app.honeybadger.io/projects/55164/check_ins
```

Comment:
Now that these check-in notifications come to Slack, should we reflect that in the documentation? Presumably the work is now less about monitoring and more about reacting to alerts on Slack.

Reply:
Added.